mobile/components/Inputs/Input.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { styled, Input as TamaguiInput, InputProps as TamaguiInputProps } from "tamagui";
+
+interface InputProps extends TamaguiInputProps {
+  // TODO: do we want to give it a placeholder?
```

Comment:
Do i need this comment? Doesn't TamaguiInputProps already have a placeholder prop?